webui/test/integration/utils/mockServer.ts

```diff
@@ -46,7 +49,7 @@ export class MockServer {
   async start(): Promise<string> {
     return new Promise(async (resolvePromise, reject) => {
       try {
-        const specPath = resolve(__dirname, '../../../../lakefs-oss/api/swagger.yml');
+        const specPath = resolve(this.lakeFSProjectRoot, 'api/swagger.yml');
```

Comment:
1. Update the description to describe this is a fix we made
2. Similar to how we resolve the static file path using `securePath` we should make sure this path is also secure.